ward/run.py

```diff
@@ -19,7 +19,8 @@
 
 
 @click.command()
-@click.option("-p", "--path", default=".", type=click.Path(exists=True), help="Path to tests.")
+@click.option("-p", "--path", nargs=0, default=".", type=click.Path(exists=True), help="Path to tests.")
```

Comment:
I missed this sentence in the click docs: "*For options, only a fixed number of arguments is supported. This can be configured by the nargs parameter.*" :(

I think the best option then would be to get rid of the `click.argument` below, and update this line as follows:

```suggestion
@click.option("-p", "--path", default=".", type=click.Path(exists=True), help="Path to tests.", multiple=True)
```

It means to pass in multiple paths you'll have to do e.g.

```
ward --path ./path1 \
     --path ./path2
```

This approach lets us maintain the default `.` path, meaning you can run all tests in the current directory just using `ward`, and it fixes the issue of being able to call without supplying a path.